examples/text-classification/run_pl_glue.py

```diff
@@ -24,6 +25,8 @@ class GLUETransformer(BaseTransformer):
     mode = "sequence-classification"
 
     def __init__(self, hparams):
+        if type(hparams) == dict:
+            hparams = Namespace(**hparams)
```

Comment:
should this change be in lightning_base? Doesn't seem glue specific!

Reply:
someone already opened an issue: https://github.com/PyTorchLightning/pytorch-lightning/issues/2385

Reply:
Yeah, as of now, this needs to be in the glue class. Otherwise `AttributeError: 'dict' object has no attribute 'task'` will occur.

Reply:
I am getting this same error when running finetune.py in examples/seq2seq/
Maybe this change should be added to lightning_base too.

Reply:
I tried moving the fix into lightning_base.py which still gives me the error, the fix maybe something else. This is just a workaround for now.

Can you try to see whether it fixes the issue if you add the code snippet to seq2seq?

Reply:
Sure, I added this code snippet in finetune.py in seq2seq, and it works for me now. Thanks!!